app/services/imports/local_authorities_service.rb

```diff
@@ -0,0 +1,26 @@
+require "csv"
+
+module Imports
+  class LocalAuthoritiesService
+    attr_reader :path, :count
+
+    def initialize(path:)
+      @path = path
+      @count = 0
+    end
+
+    def call
+      CSV.foreach(path, headers: true) do |row|
+        LocalAuthority.upsert(
+          { code: row["code"],
+            la_name: row["la_name"],
+            start_date: Time.zone.local(row["start_year"], 4, 1),
+            end_date: (Time.zone.local(row["end_year"], 3, 31) if row["end_year"]),
+            previous_location_only: row["previous_location_only"] || false },
```

Comment:
What does this field represent?

Reply:
Well, some of the fields are non England as we discussed earlier, so they shouldn't be shown in current la questions but should be shown in previous la questions. I figured marking them in the db would be the easiest way to filter them out when needed